src/pages/LenderProfile/LenderProfile.vue

```diff
@@ -33,6 +43,55 @@ export default {
 		LenderProfileWrapper,
 		NotFoundWrapper,
 	},
+	setup() {
```

Comment:
I see this is `setup` within an older Vue component, does `onMounted` and everything work OK?